requirements/edx/edx-private.txt

```diff
@@ -18,3 +18,6 @@
 # This repository contains schoolyourself-xblock, which is used in
 # edX's "AlgebraX" and "GeometryX" courses.
 -e git+https://github.com/schoolyourself/schoolyourself-xblock.git@5e4d37716e3e72640e832e961f7cc0d38d4ec47b#egg=schoolyourself-xblock
+
+# Peer instruction xblock
+ubcpi-xblock==0.4.2
```

Comment:
I have 0.4.2 on pypi. Forgot to tag for 0.4.2 on github. Now it's on github. 
So we prefer github over pypi?